website/blog/2020-10-27-ld-scores.md

```diff
@@ -0,0 +1,28 @@
+---
+id: ldscore-comparisons
+title: LD score comparisons between gnomAD and UK Biobank
+author: Rahul Gupta, on behalf of the Pan UKBB Team
+tags: [LD, scores]
+---
+
+In this analysis, we aimed to better understand the similarities and differences between imputation-based and sequence-based LD scores. To this end, we compared LD scores from gnomAD (sequence-based) and UKB (imputation-based) across four ancestry groups: African (`AFR`), Admixed American (`AMR`),  East Asian (`EAS`), and non-Finnish European (`EUR`; `NFE` in gnomAD). This also serves as a more general comparison between LD scores from two separate cohorts. LD scores were obtained as follows.
```

Comment:
Oh I hadn't realized you were comparing to `NFE` - can you try `NWE` and see if that does any better?

Reply:
@rahulg603 Would you mind checking this? I think you can commit to the `ld_docs` branch to reflect your changes.
(this is stacked on #24 and includes your commits)

Reply:
You were right @konradjk, NWE does do a bit better!